Working

Pilad opened the pull request:
Фикс для дизассемблера. В конфиге можно вернуть старую механику по желанию.
Мехи сделанные из разных микросхем стакаются
Верстаки форсетри и НЕИ в мэ/верстаке начинают исправно выкладывать рецепты с роборуками моторами и т д
Дизассемблер теперь при разборке возвращает микрухи, самые низкие по техпроцессу (то есть HV микруху можно получить на разных техпроцессах (красный, желтый и т д), вне зависимости от использованной при крафте меха дизассемблер вернет красную.

Pilad committed:
Update GT_Mod.java

Забыл дописать импорт.

Pilad committed:
Update GT_Mod.java

Pilad committed:
Update IGT_RecipeAdder.java

Pilad committed:
Update GT_Recipe.java

Pilad committed:
Update GT_Shaped_Recipe.java

Pilad committed:
Update GT_Proxy.java

Pilad committed:
Update GT_RecipeAdder.java

Pilad committed:
Update GT_Recipe.java

Pilad committed:
Update GT_Shaped_Recipe.java

Pilad committed:
Update GT_MetaTileEntity_Disassembler.java

Переработан дизассемблер. Возвращает микрухи при раскрафчивании мехом самого первого тира из данного техпроцесса. Убрано длинное нбт, вследствии чего мехи скрафченные из разных элементов не стакались. По шифтклику теперь компоненты в верстаке будут выкладываться, а не как раньше.

Pilad committed:
Update Materials.java

Айди подставить свободные, если будет конфликт.Отбалансить у топлива калорийность в реалиях Грега. Скорее всего будет ошибка о том, что код слишком велик в этом классе. Нужно решить проблему с добавлением большего числа материалов.

Pilad committed:
Update ProcessingLog.java

Добавлено получение древесного газа(Coalgas) через пиролизку.

Pilad committed:
Update GT_MachineRecipeLoader.java

Собственно химия на получение высококалорийного топлива.
Так же вписан код на разбор пылек , хотя можно сделать это автоматически через класс материалов, я вписал в материалах код и закомментировал,например : LithiumHydroxide - /, 1, Arrays.asList(new MaterialStack(Oxygen, 1), new MaterialStack(Lithium, 16), new MaterialStack(Water, 1))/ . Если логичнее через класс материалов авторазбор сделать,чтоб грег сам генерил рецепты, то в этом классе удалить вручную вписанные рецепты.

Pilad committed:
Update GT_MetaTileEntity_Hatch_OutputBus.java

Автоматическое извлечение предметов для аутпут баса.

Pilad committed:
Update GT_MetaGenerated_Tool.java

Добавляет в креатив все инструменты , роторы и т.д с прочностью из материала Нейтроний.

Pilad committed:
Update GT_Loader_OreDictionary.java

Добавление ордикта удобрению форестри и индастриала.

Pilad committed:
Update GT_Proxy.java

При запуске клиента и фазе загрузки материалов ничего не не писалось в лог или в консоли клиента. так будет выводить текущий прогресс в процентах вероятно от 0 до 100. Baking : 0 --> Baking : 100

Pilad commented:
Добавил еще немного изменений. У измененных файлов есть небольшое описание,что изменение делает. Обратите внимание на класс материалов. Там будет проблема с размером кода больше 64кб и коррекцией свободных айди.(если капсулы ненужны , то айди установить в -1) Так же может быть проблема в том, что у грега всего 1000 айди для материалов. Может стоит расширить до 5к например? У меня знаний не хватает, как эти две проблемы решить.

Serious07 commented:
> Дизассемблер теперь при разборке возвращает микрухи, самые низкие по техпроцессу (то есть HV микруху можно получить на разных техпроцессах (красный, желтый и т д), вне зависимости от использованной при крафте меха дизассемблер вернет красную.

Я бы хотел чтобы была возможность возвращать старый вариант в конфиге грега.

Pilad commented:
> > Дизассемблер теперь при разборке возвращает микрухи, самые низкие по техпроцессу (то есть HV микруху можно получить на разных техпроцессах (красный, желтый и т д), вне зависимости от использованной при крафте меха дизассемблер вернет красную.
> 
> Я бы хотел чтобы была возможность возвращать старый вариант в конфиге грега.

Так и есть,есть опция в конфиге. Там выше я про это написал: В конфиге можно вернуть старую механику по желанию.

Pilad committed:
Update GT_MetaTileEntity_Pump.java

Теперь помпа будет сама выталкивать жижу вверх по дефолту.

Pilad committed:
Update GT_MetaTileEntity_SteamTurbine.java

Если паровая турбина улв, то буфер превращается в 0. И турбина просто не пашет.

Pilad commented:
Фикс улв турбины добавил. Забыл буфер привести к нужному размеру,чтоб остался на том же уровне. Пофиксил следующим ПРом.

Pilad committed:
Update GT_MetaTileEntity_SteamTurbine.java

Чтоб буфер остался на том же уровне.
